### PR TITLE
EVG-13011 support custom commit title/message for pr commit queue

### PR DESCRIPTION
**Pull request by john-m-liu**
now passing -t/--title in the merge comment allows you to specify the commit title, and passing -m/--message allows you to specify the commit message

even though the data structure supports it, I didn't add a way to specify a title/message for modules because the syntax for that would be really weird and I don't see a use case

### CODE REVIEW

**Review by ablack12 — CHANGES_REQUESTED** [review]
I think if the title is overridden then that's the title that should be used in the version
https://github.com/evergreen-ci/evergreen/blob/02729cf1cba92556f09bfd73747fd90987327de4/model/patch/patch.go#L711

**Comment by john-m-liu** [comment]
I somewhat disagree - having the boilerplate description helps identify that the patch was created by the commit queue. That way if someone is going through their patches later, they won't wonder what this is since they didn't create the patch

**Comment by ablack12** [comment]
> I somewhat disagree - having the boilerplate description helps identify that the patch was created by the commit queue. That way if someone is going through their patches later, they won't wonder what this is since they didn't create the patch

Sorry, I didn't mean that the "Commit Queue Merge" part should be dropped. Specifically the part where PR.GetTitle() is passed into the template. This should match the title that they intend to merge.

**Comment by john-m-liu** [comment]
ah I see, fixed

File: makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildDir := bin
 tmpDir := $(abspath $(buildDir)/tmp)
 nodeDir := public
 packages := $(name) agent agent-command agent-util agent-internal apimodels operations cloud cloud-userdata db util plugin units graphql
-packages += thirdparty thirdparty-docker auth scheduler model validator service repotracker
+packages += thirdparty thirdparty-docker auth scheduler model validator service repotracker cmd-codegen-core
```
Comment (ablack12) [inline makefile:7]: sneaky haha

**Review by ablack12 — APPROVED** [review]

**Pull request merged**